README.md

```diff
@@ -1,57 +1 @@
-# Super-List
-
-The learning goals of this project are to practice using ES6 standards in Javascript and to implement Webpack.  This project is recreating part of a, now defunct, todolist app called minimalist.
-
-## Buit With
-
-- HTML
-- CSS
-- Javascript
-- Webpack
-
-## Live Demo
-
-[Super List](https://crclothier.github.io/Super-List/)
-
-## Getting Started
-
-To get a local copy up and running follow these simple example steps:
-
-### Prerequisites
-
-Basic knowledge of:
-- HTML, CSS & Javascript
-- Git & Github
-
-### Setup
-
-- Create a directory on your local computer.
-- Use git init in that directory.
-- Clone this repository into your folder.
-
-### Usage
-
-Edit the files to practice professional standards.
-
-## Authors
-
-**Chris Clothier**
-GitHub: [@CRClothier](https://github.com/crclothier)  
-LinkedIn: [Chris Clothier](https://www.linkedin.com/in/crclothier/)
-
-## 🤝 Contributing
-
-Feedback, issues, and feature requests are welcome!
-
-## Show your support
-
-Give a star ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Thanks to Microverse for their README template.
-- Thanks to my learning partners for unblocking me time and time again!
-
-## 📝 License
-
-This project is [MIT](MIT.md) licensed.
+# Super-List
```

Comment:
- [ ] **[REQUIRED]** Please consider adding a descriptive README in this current branch as it is a requirement to have a descriptive README in each branch you created. I can see you deleted them, please bring them back 😉